com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/jobs/AbortedOperationsCleaner.java

```diff
@@ -56,18 +51,7 @@ public void execute(Date expirationTime) {
     }
 
     private boolean isInActiveState(String operationId) {
-        Operation operation = getOperation(operationId);
-        return operation != null && operation.getState() == null;
-    }
-
-    private Operation getOperation(String operationId) {
-        try {
-            return operationService.createQuery()
-                                   .processId(operationId)
-                                   .singleResult();
-        } catch (NoResultException e) {
-            return null;
-        }
+        return flowableFacade.getProcessInstance(operationId) != null;
```

Comment:
I had checked code in flowable and it will throw exception if result from sql query is more than 1 entry. If the result is empty, it will return null.